packages/client/utils/GitHubManager.ts

```diff
@@ -23,6 +23,8 @@ type DocVariables = any
 
 abstract class GitHubManager {
   static SCOPE = 'admin:org_hook,read:org,repo,user:email,write:repo_hook'
+  // static SCOPE = 'admin:org_hook,read:org,repo,user,write:repo_hook'
```

Comment:
how excited are you that we didn't have to change scopes to make this request :pray:

Reply:
😬   I did have to change the scope unfortunately! I think you're looking at a previous commit. I left a comment in the `GitHubManager` file mentioning that I had to change the `user` scope so that we have access to the private contributions